utils/goDotEnv.go

```diff
@@ -0,0 +1,19 @@
+package utils
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func GoDotEnvVariable(key string) string {
+
+	err := godotenv.Load(".env")
```

Comment:
The function will load variables from the `.env` file every time you call it, and then it will send you the value by searching from env. In production, where we won't be using the `.env` file and try to get 5 variables from the environment by calling the function `getDotEnvVariable` will give errors 5 times as there is no `.env` file to load.

A better approach will be to call this function once and the env will be populated then we can use `os.Getenv()` (better to use `os.LookupEnv()` ) in the actual code rather than calling this variable.

To ensure this you can rename this function to `setUpEnv` and call it in the `init` function so that you are sure of its running. and try to keep a check on the environment if it is production then there should be no need to call the Load function to load the env.

Reply:
Just to check if my understanding is correct, for the `DEVELOPMENT` env we call the above function `setUpEnv` and load the `.env` file and use `os.LookupEnv()` wherever it has to be used.

In case of production env as we store the env values in github secrets, there will be no `.env` file, so will directly `os.LookupEnv()` fetch the env variables?

Reply:
yes.

Reply:
+1
as Ritik suggested, this should only be for the development

Reply:
Sure creating a ticket for the production setup and we can do that after the APIs, I think.

@RitikJaiswal75 @prakashchoudhary07